test/successful-directories.test.js

```diff
@@ -1027,20 +1027,20 @@ describe('works fine if sync loader returns a Promise from a JS file', () => {
   const startDir = temp.absolutePath('a/b/c/d/e/f');
   beforeEach(() => {
     temp.createFile(
-      'a/b/c/d/e/f/foo.config.js',
+      'a/b/c/d/e/f/bar.config.js',
```

Comment:
These tests kept failing for me, semi-randomly, and this was quicker than figuring out what module cache issues were to blame...

Reply:
This was, as I explained, necessary to make the test suite pass at all. The snapshots kept breaking, returning an object instead of a Promise, exactly like a cached search would return the eventually-resolved config object for a duplicate path.

This test isn’t testing the cache, so this is an appropriate change, in my opinion.

Reply:
@chrisblossom I opened a follow-up issue about this: https://github.com/davidtheclark/cosmiconfig/issues/148